data/json/items/gun/50.json

```diff
@@ -67,7 +67,9 @@
     },
     "material": [ "steel", "wood" ],
     "clip_size": 1,
+    "modes": [],
```

Comment:
> Drop as in delete it? Because that crashes my game.

JSON changes should never crash the game. If you can factor that into a minimal test case I'll look into it.